cli/gluster-block.c

```diff
@@ -254,9 +254,9 @@ glusterBlockCliRPC_1(void *cobj, clioperations opt)
     }
   } else if (errMsg[0]) {
     LOG("cli", GB_LOG_ERROR, "%s", errMsg);
-    MSG(stderr, "%s\n", errMsg);
+    MSG(stderr, "%s", errMsg);
```

Comment:
No need of '%s' in MSG() according to your change, I see you just removed '\n' in many more places, please remove %s in all of them.

Reply:
Sorry, I may not understand this.

BTW, we are also having this:

./utils/common.c:97:        MSG(stderr, "minimum acceptable block size is %d bytes\n", GB_DEFAULT_SECTOR_SIZE);

And the 'errMsg' is the variable argument, not the format string, as far as I know only the following type we could remove the "%s":

MSG(stderr, "%s", "blabla......");  --> MSG(stderr, "blabla....");  // has fmt without any variable argument.

Make sense ?

Thanks.
BRs

Reply:
Sorry I have commented at a wrong line.